recipes/kafka-python-ng/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "kafka-python-ng" %}
+{% set version = "2.0.3" %}
+{% set bundle = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash = "b447ce219097e27741a46c9d1ee5cb55f42974336277bf41a456bba7623e3078" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ bundle }}
+  url: https://github.com/wbarnha/{{ name }}/archive/v{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: {{ build }}
+  noarch: python
+  script:
+    - "{{ PYTHON }} setup.py bdist_wheel --dist-dir=dist"
+    - "{{ PYTHON }} -m pip install --find-links=dist --no-deps --ignore-installed --no-cache-dir -vvv {{ name }}"
+
+
+requirements:
+  build:
+    - python
+    - git
+    - pip
+    - setuptools
+    - setuptools_scm
```

Comment:
Based on [this section of their knowledge base](https://conda-forge.org/docs/maintainer/knowledge_base/#using-setuptools_scm), this needs to be under the `host` section.

They also mention using the PyPi tarball as a source since it has metadata info included. That might simplify the overall process.

[Another relevant section](https://conda-forge.org/docs/maintainer/knowledge_base/#placing-requirements-in-build-or-host):
> The rule of the thumb is:
> * If it needs to run during the build, it goes in `build`.
> * If it needs to be available on the target host, it goes in `host`.
> * If both conditions are true, it belongs in both.